<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managerDialog</class>
 <widget class="QDialog" name="managerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>831</width>
     <height>561</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page1">
    <widget class="QGroupBox" name="box1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>331</width>
       <height>481</height>
      </rect>
     </property>
     <property name="title">
      <string>工作人员</string>
     </property>
     <widget class="QTableWidget" name="waiterTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="chefTable">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Page 1</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="box2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>351</width>
       <height>481</height>
      </rect>
     </property>
     <property name="title">
      <string>菜品反馈</string>
     </property>
     <widget class="QTableWidget" name="dishTable">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>301</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page2">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Page 2</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="box3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>561</width>
       <height>511</height>
      </rect>
     </property>
     <property name="title">
      <string>当前流水</string>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>491</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="refreshBtn">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>140</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
    <widget class="QLabel" name="movieLabel">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>280</y>
       <width>161</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="box4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>641</width>
       <height>461</height>
      </rect>
     </property>
     <property name="title">
      <string>流水账</string>
     </property>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string> 切换</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
